zubhub_frontend/zubhub/src/assets/js/styles/views/page_wrapper/pageWrapperStyles.js

```diff
@@ -10,7 +10,7 @@ const styles = theme => ({
     fontSize: '1.15rem',
   },
   mainContainerStyle: {
-    maxWidth: '2000px',
+    maxWidth: '6000px',
```

Comment:
This seems to me is the only required change

Reply:
Yes it was needed so that it could fit all the screen sizes